attributes/ernest.rb

```diff
@@ -42,8 +44,13 @@
                                            'firewall-updater-vcloud-connector', 'nat-updater-vcloud-connector',
                                            'router-deleter-vcloud-connector', 'network-deleter-vcloud-connector']
 
+=begin
```

Comment:
you've removed all stores from ruby-based deploys but you ddin't added them to to the gpb? they are not going to be deployed...